x-pack/plugins/security_solution/cypress/e2e/exceptions/add_edit_flyout/flyout_validation.cy.ts

```diff
@@ -311,6 +320,39 @@ describe('Exceptions flyout', () => {
     closeExceptionBuilderFlyout();
   });
 
+  it('Warns users about mapping conflicts on problematic field selection', async () => {
+    // open add exception modal
+    openExceptionFlyoutFromEmptyViewerPrompt();
+
+    // find 'doc_id' field which has mapping conflicts accross different indices
+    cy.get(FIELD_INPUT).eq(0).click({ force: true });
```

Comment:
Please extract these to lines into a task method :)

Reply:
done